looper/utils.py

```diff
@@ -0,0 +1,157 @@
+""" Helpers without an obvious logical home. """
+
+from collections import defaultdict
+import logging
+import os
+import yaml
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+
+class CommandChecker(object):
```

Comment:
I'm not suggesting nesting in another class, I was thinking of the module level. by *with* I meant *parallel to* . right now we're not even using this yet, right? it's still just a concept...